.eslintignore

```diff
@@ -2,3 +2,6 @@
 flow-typed
 lib/vendor
 lib/utils/parseCalcExpression/parser.js
+
+# Unignore config files like .prettierrc.js, because they're ignored by default
+!.*rc.js
```

Comment:
Have you got a reference for this? I would be interested to have a read on why ESLint chooses to ignore these by default 🤔

Reply:
Nope, and I can't find default ignore list on ESLint website as well. I faced this few years ago for the first time. Found solution, and now copied this solution to stylelint.

Reply:
https://github.com/eslint/eslint/blob/7621f5d2aa7d87e798b75ca47d6889c280597e99/lib/cli-engine/ignored-paths.js#L34 They ignore dotfiles by default.